packages/python/patch-addrinfo_sa_data

```diff
@@ -0,0 +1,19 @@
+--- Modules/socketmodule.c
++++ Modules/socketmodule.c
+@@ -1380,10 +1380,16 @@
+     default:
+         /* If we don't know the address family, don't raise an
+            exception -- return it as an (int, bytes) tuple. */
++#ifndef __CloudABI__
+         return Py_BuildValue("iy#",
+                              addr->sa_family,
+                              addr->sa_data,
+                              sizeof(addr->sa_data));
++#else
++        /* CPython assumes addrinfo.sa_data is declared as char[n], for some
++           finite n, but cloudlibc declares addrinfo.sa_data as char[] */
```

Comment:
That's interesting that Python makes use of this. I'm also quite torn what to do here. Should we declare our `sa_data[]` to also have some random size? Maybe Python should be patched up to not use it at all, even on other platforms?